docs-source/spring/content/development/cli/_index.md

```diff
@@ -7,35 +7,50 @@ microservices applications as well as bindings with the resources that they use.
 Download the CLI [here](https://github.com/oracle/microservices-datadriven/releases/tag/OBAAS-1.0.0).
 The platform-specific binary can be renamed to `oractl` for convenience.
 
+Table of Contents:
+
+* [Using the CLI](#using-the-cli)
+* [Available Commands](#available-commands)
+* [Logging Information](#logging)
+  * [Help](#help)
+  * [Connect to the backend](#connect)
+  * [Create application namespace](#create)
+  * [Delete application namespace](#delete)
+  * [Bind database schema/kubernetes secrets](#bind)
+  * [Deploy a service](#deploy)
+  * [List - show details about deployments](#list)
+  * [Manage config server data](#config)
+  * [GraalVM compile commands](#compile)
+
 ## Using the CLI
 
 1. Expose the Oracle Backend for Spring Boot and Microservices Admin server that the CLI calls using this command:
 
     ```cmd
-    kubectl port-forward services/obaas-admin -n obaas-admin 8080:8080
+    kubectl port-forward services/obaas-admin -n obaas-admin 8080
     ```
 
-2. Start the CLI in interactive mode by running `oractl` from your terminal window. For example:
+1. Start the CLI in interactive mode by running `oractl` from your terminal window. For example:
 
     ```cmd
     oractl
     ```
 
 ## Available Commands
 
+### help
+
 Short descriptions for the available commands can be viewed by issuing the `help` command and detailed help for any individual
 commands can be viewed by issuing `help [command-name]`. For example:
 
 ```cmd
-oractl:>help
 AVAILABLE COMMANDS
 
 Admin Server Commands
-       connect: Connect to the Oracle Backend Administration Service.
+       connect: Connect to the OBaaS Spring Cloud admin console.
 
 Application/Namespace Commands
        create: Create an application/namespace.
-       delete: Delete a entire application/namespace or service.       
 
 Built-In Commands
        help: Display help about available commands
```

Comment:
To be consistent, add a period.

Reply:
This is in the source code. I'll fix it there